fundingmanager.go

```diff
@@ -2483,8 +2497,9 @@ func (f *fundingManager) newChanAnnouncement(localPubKey, remotePubKey,
 		ChainHash:       chainHash,
 		Timestamp:       uint32(time.Now().Unix()),
 		ChannelFlags:    chanFlags,
-		TimeLockDelta:  uint16(f.cfg.DefaultRoutingPolicy.TimeLockDelta),
+		MsgFlags:        1,
```

Comment:
This should be extracted into a describing constant  :p

Reply:
fixed